charts/operator/templates/aggregate-roles.yaml

```
@@ -0,0 +1,246 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "osac-operator.fullname" . }}-clusterorder-editor
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)